### PR TITLE
Added Favicon

### LINKED ISSUE #288: Add Favicon
Currently, there is no favicon on my site. Need to add it in `head` tag

### PR DESCRIPTION
**Pull request by ahmed-sudani**
## Issue
Closes #288 

## What is the change?
Added a favicon

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <title>My Site | Real Dev Squad</title>
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" type="image/x-icon" href="/icons/favicon.ico" />
```
Comment (rohan09-raj) [inline app/index.html:9]: Does the `favicon.ico` file exist? or even the icons folder exists in the code?

Comment (ahmed-sudani) [reply app/index.html:9]: @rohan09-raj Yes, I have added the favicon.ico and the icons folder

Comment (rohan09-raj) [reply app/index.html:9]: Ohh ok, sorry I missed it. Thanks I can see it now

**Review by rohan09-raj — APPROVED** [review]
Thanks for adding the favicon. Approving and merging the pull request

**Pull request merged**